On macos 10.15:

```
builder for '/nix/store/269qfdl1bs0glqhkaz6miqzjvn39q3sb-python3.9-xknx-0.18.6.drv' failed with exit code 1; last 10 log lines:
          assert binaryinput.state
          await asyncio.sleep(reset_after_sec * 1.2)
  >       assert not binaryinput.state
  E       assert not True
  E        +  where True = <xknx.devices.binary_sensor.BinarySensor object at 0x10c6800a0>.state
  
  test/devices_tests/binary_sensor_test.py:95: AssertionError
  =========================== short test summary info ============================
  FAILED test/devices_tests/binary_sensor_test.py::TestBinarySensor::test_process_reset_after
  ======================= 1 failed, 1639 passed in 11.94s ========================
```

--- a/pkgs/development/python-modules/xknx/default.nix
+++ b/pkgs/development/python-modules/xknx/default.nix
@@ -11,14 +11,14 @@
 
 buildPythonPackage rec {
   pname = "xknx";
-  version = "0.18.6";
+  version = "0.18.7";
   disabled = pythonOlder "3.7";
 
   src = fetchFromGitHub {
     owner = "XKNX";
     repo = pname;
     rev = version;
-    sha256 = "sha256-LyeEEynfgB3juprFeiIOFvFXWtTiGbZ8D9m6SnpD0ws=";
+    sha256 = "17w6a4ci4w6ggxpa99197f84awd116ygzd3fa5cvn1a7221dvdj4";
   };
 
   propagatedBuildInputs = [
@@ -43,5 +43,6 @@ buildPythonPackage rec {
     homepage = "https://github.com/XKNX/xknx";
     license = with licenses; [ mit ];
     maintainers = with maintainers; [ fab ];
+    platforms = platforms.linux;
   };
 }

